.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   env: {
     mocha: true,
     node: true,
+    es6: true,
```

Comment:
This is so it doesn't throw an error when using "new" features from ES6 (e.g. `new Set()`). It is not really needed for this PR, but is a good and simple addition nevertheless.